### PR TITLE
Add category title to the main forum page if category is selected

### PR DESCRIPTION
**Pull request by valentinludu**
https://app.charmverse.io/charmverse/page-8704528490535499

### CODE REVIEW

File: hooks/useForumCategories.ts [hunk 1]
```diff
@@ -7,6 +8,8 @@ import { useCurrentSpace } from './useCurrentSpace';
 
 export function useForumCategories() {
   const currentSpace = useCurrentSpace();
+  const router = useRouter();
```
Comment (mattcasey) [inline hooks/useForumCategories.ts:11]: This has bit us in the past (putting state into a data source hook) I think we should use a different hook if you want to capture the 'current category'.

**Review by mattcasey — CHANGES_REQUESTED** [review]
remove the logic to determine current category from the useForumCategories hook. it won't always be defined by the URL, i think its safer to just let components maintain that logic

File: hooks/useForumCategories.ts (hunk 1 above)
Comment (valentinludu) [reply hooks/useForumCategories.ts]: OK. Then I will keep it as before. Outside, each component will determine how it gets the id.

**Comment by valentinludu** [comment]
> remove the logic to determine current category from the useForumCategories hook. it won't always be defined by the URL, i think its safer to just let components maintain that logic

Done

**Review by mattcasey — APPROVED** [review]

**Pull request merged**